rollingpin/args.py

```diff
@@ -81,6 +81,15 @@ def _add_iteration_arguments(config, parser):
         dest="pause_after",
     )
 
+    iteration_group.add_argument(
+        "--timeout",
+        default=None,
```

Comment:
It should happen at this level, like [sleeptime](https://github.com/dellis23/rollingpin/blob/cb4e9a78cf06a0eee50de0014185a4f756a12220/rollingpin/args.py#L64) does so that `--help` shows the defaults that would be used.